nncf/sparsity/magnitude/algo.py

```diff
@@ -63,6 +67,15 @@ def set_sparsity_level(self, sparsity_level):
 
         self.threshold = self._select_threshold()
         self._set_masks_for_threshold(self.threshold)
+        self.run_batchnorm_adaptation()
+
+    def run_batchnorm_adaptation(self):
+        initializer_params = self.config.get("initializer", {})
```

Comment:
why not to move it to BaseAlgoController class, if this initializer is common for all algorithms?

Reply:
Agreed. Moved to the base `CompressionAlgorithmController`